paper/src/main/java/vg/civcraft/mc/citadel/listener/ModeListener.java

```diff
@@ -248,6 +249,11 @@ public void handleInteractBlock(PlayerInteractEvent e) {
 					sb.append(String.format("%sAcid block mature in %s", ChatColor.YELLOW, TextUtil.formatDuration(remainingTime, TimeUnit.MILLISECONDS)));
 				}
 			}
+			if (e.getPlayer().hasPermission("citadel.admin")) {
+				sb.append(String.format("\n%sActivityMap.Activity = %s",
+						ChatColor.WHITE,
```

Comment:
Recommend usage of Kyoris Components over BungeeCords ChatColor, just since ChatColor is deprecated in paper

Reply:
Could switch the ChatColor import from Bungee to Bukkit since the latter isn't deprecated.

Reply:
I didn't do this because it means I need to rework the whole function which is huge and requires additional testing.
As Protonull suggested ChatColor from Bukkit is not yet deprecated.

I propose to make this refactoring as an additional task.